### PR TITLE
Change to add information about where to store architecture diagrams

### PR DESCRIPTION
**Pull request by pierssinclairssw**
Hi @wicksipedia , @william-liebenberg ,

1. Please review the GitHub and Azure DevOps changes.

Note: this rule is in relation to Architecture Diagram storage. We will have another rule which goes into the specifics of documentation (and link to Diagrams). @calumjs , myself and @tiagov8 are working on this.

### CODE REVIEW

**Review by wicksipedia — COMMENTED** [review]
Hi Piers
We already have some rules that cover these items generally: https://www.ssw.com.au/rules/awesome-documentation
I'm not a fan of the different recommended approaches depending on the ALM environment.
1. I'd mention that the diagram should be referenced (or linked to) in the README file
2. The recommended approach is to have a `/docs` folder and use that to store the wiki pages.
3. The non-recommended approaches can then be Azure DevOps/GitHub specific

**Comment by pierssinclairssw** [comment]
> 2. The recommended approach is to have a `/docs` folder and use that to store the wiki pages.

@william-liebenberg 

1. Can you confirm that this is also the recommended approach for Azure DevOps rather than using the wiki? My understanding was that you prefer the wiki but having it stored in the repo.

File: rules/architecture-diagram/rule.md [hunk 1]
```diff
@@ -164,6 +164,19 @@ Maintain standards to keep your diagrams consistent:
 ![Figure: Good Example - SSW People (a Static Site - Gatsby and React with Dynamics 365 and SharePoint Online) - you can just as easily create colorful, engaging diagrams suitable for all of your project stakeholders](SSW.People-Architecture-Diagram.png)
 :::
 
+### Tip 9: Tip 9 – Where to store Diagrams?
+
+Standardizing where your organisation stores architecture diagrams is important to ensure a consistent experience among developers. There are a few options depending on which source control you use:
+
```
Comment (william-liebenberg) [inline rules/architecture-diagram/rule.md:170]: As per our rule (https://www.ssw.com.au/rules/awesome-documentation) the project architecture image is usually in the README.md file which will be located in root folder of the repo.

The rest of the documentation goes into a `/docs` folder - regardless of which Source Control is being used.

When using Azure DevOps I prefer the coded wiki which gets pulled down with the code from the repository. The other option is to use the built-in wiki that does not get pulled down with the code from the repository.

When using GitHub, there is also a separate Wiki repo for your project. 
For instance to clone your wiki to your local machine you can do: `git clone https://github.com/YOUR_USERNAME/YOUR_REPOSITORY.wiki.git`
Notice the `.wiki.` part of the git link.

**Comment by william-liebenberg** [comment]
> My understanding was that you prefer the wiki but having it stored in the repo.

Yes i prefer to use the coded wiki for all the wiki pages. As for the architecture diagram, it should be in the project `README.md` file.

**Comment by pierssinclairssw** [comment]
As per my conversation with @wicksipedia and @bradystroud , I will update this rule and https://www.ssw.com.au/rules/awesome-documentation to have /docs as the default location for all documents.

The wiki will be a separate recommendation and we will suggest that it should be built off the /docs folder.

**Comment by pierssinclairssw** [comment]
Hi @wicksipedia , @william-liebenberg ,

Please review the new changes and let me know your thoughts.

-Piers

**Review by wicksipedia — APPROVED** [review]
I made a small update to the recommended GitHub approach (using pages is great but may not be useful for every project)

**Review by william-liebenberg — APPROVED** [review]
Lots of changes, but looks good :)

**Review by adamcogan — APPROVED** [review]
Ready for the Chewing The Fat !

**Pull request merged**